system_tests/espresso_e2e_test.go

```diff
@@ -538,6 +516,50 @@ func TestEspressoE2E(t *testing.T) {
 
 	checkStaker := os.Getenv("E2E_CHECK_STAKER")
 	if checkStaker == "" {
+		log.Info("Checking the escape hatch")
+		// Start to check the escape hatch
+		address := common.HexToAddress(lightClientAddress)
+
+		txOpts := builder.L1Info.GetDefaultTransactOpts("Faucet", ctx)
+		// Freeze the l1 height
+		err := lightclientmock.FreezeL1Height(t, builder.L1.Client, address, &txOpts)
+		Require(t, err)
+
+		err = waitForWith(t, ctx, 1*time.Minute, 1*time.Second, func() bool {
+			isLive, err := lightclientmock.IsHotShotLive(t, builder.L1.Client, address, uint64(delayThreshold))
+			if err != nil {
+				return false
+			}
+			return !isLive
+		})
+		Require(t, err)
+
+		// Wait for the switch to be totally finished
+		currMsg, err := node.ConsensusNode.TxStreamer.GetMessageCount()
+		Require(t, err)
+		validatedMsg := arbutil.MessageIndex(0)
```

Comment:
Why are we checking the first message here?

Reply:
Thanks. Just want to declare a variable here. Have changed to `var validatedMsg arbutil.MessageIndex`. This would be better